Content.Server/Chemistry/Components/SolutionRegenerationComponent.cs

```diff
@@ -14,31 +14,30 @@ public sealed partial class SolutionRegenerationComponent : Component
     /// <summary>
     /// The name of the solution to add to.
     /// </summary>
-    [DataField("solution", required: true), ViewVariables(VVAccess.ReadWrite)]
+    [DataField("solution", required: true)]
     public string SolutionName = string.Empty;
 
     /// <summary>
     /// The solution to add reagents to.
     /// </summary>
-    [DataField("solutionRef")]
-    public Entity<SolutionComponent>? Solution = null;
+    public Entity<SolutionComponent>? SolutionRef = null;
```

Comment:
This needs to be an EntityUid as it will break with save + load.

Reply:
Added em back. Im gonna remove the fix flag too since I couldn't replicate it and dunno if this fixed anything.